cypress/integration/portal-dashboard/portal-dashboard-header.spec.js

```diff
@@ -35,56 +35,101 @@ context("Portal Dashboard Header", () => {
         cy.get("[data-cy=last-run-header]").should("be.visible");
         cy.get("[data-cy=last-run-row]").should("be.visible").and("have.length", 6);
         cy.get("[data-cy=header-menu]").click();
-        cy.get("[data-cy=last-run-menu-item]").should("be.visible").click();
+        cy.get("[data-cy=menu-list]").should("be.visible");
+        cy.get("[data-cy=last-run-menu-item]").click();
+        cy.get("[data-cy=menu-list]").should("not.be.visible");
+        cy.wait(1000);
         cy.get("[data-cy=last-run-header]").should("not.exist");
         cy.get("[data-cy=last-run-row]").should("not.exist");
       });
       it("verify we can show Last Run column", () => {
         cy.get("[data-cy=header-menu]").click();
-        cy.get("[data-cy=last-run-menu-item]").should("be.visible").click();
+        cy.get("[data-cy=menu-list]").should("be.visible");
+        cy.get("[data-cy=last-run-menu-item]").click();
+        cy.get("[data-cy=menu-list]").should("not.be.visible");
         cy.get("[data-cy=last-run-header]").should("be.visible");
         cy.get("[data-cy=last-run-row]").should("be.visible").and("have.length", 6);
       });
       it("verify 'Hide Last Run column' option's selected state is maintained across all views", () => {
+        // First hide the column
         cy.get("[data-cy=header-menu]").click();
-        cy.get("[data-cy=last-run-menu-item]").should("be.visible").click();
+        cy.get("[data-cy=menu-list]").should("be.visible");
+        // Using force: true because the menu item might be covered by other elements or have pointer-events disabled
+        cy.get("[data-cy=last-run-menu-item]").click({ force: true });
+        // Wait for the state to update and the column to be hidden
```

Comment:
Are we actually waiting here? I may be misunderstanding, but it doesn't look like we wait. Maybe this comment isn't necessary?